ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -1803,7 +1805,10 @@
     "AccessControl": [
       {
         "UserRoles": [
-          "[RESPONDENTSOLICITORTWO]"
+          "[RESPONDENTSOLICITORONE]",
+          "[RESPONDENTSOLICITORTWO]",
```

Comment:
"[RESPONDENTSOLICITORONE]","[RESPONDENTSOLICITORONESPEC]" should be removed


I'm not sure about "[RESPONDENTSOLICITORONESPEC]" though?

Reply:
1v2 spec same solicitor still needs both 1 and 2 fields, that's why respondentsolicitoronespec is needed.
I remove the unspec ones